flow/src/main/java/com/sequenceiq/flow/service/FlowService.java

```diff
@@ -276,7 +305,7 @@ public Page<FlowCheckResponse> getFlowChainsByChainIds(List<String> chainIds, Pa
     }
 
     private Map<String, List<FlowLogWithoutPayload>> fetchFlowLogsByParentChainId(Set<String> flowChainIdsSet,
-            Map<String, String> parentChainIdByChainIdMap) {
+                                                                                  Map<String, String> parentChainIdByChainIdMap) {
```

Comment:
Strange indentation.

Reply:
Sorry about this - the reformat tool automatically changes this. Updated this back!